src/transaction/locator_sr.c

```diff
@@ -13723,11 +13723,24 @@ xlocator_demote_class_lock (THREAD_ENTRY * thread_p, const OID * class_oid, LOCK
   return lock_demote_class_lock (thread_p, class_oid, lock, ex_lock);
 }
 
-static bool
+bool
 locator_is_BU_locked (const OID & class_oid)
```

Comment:
I think we can replace it with `lock_has_lock_on_object` that is a general function for the purpose.